Platform/TigerlakeBoardPkg/CfgData/CfgDataExt_Upx11.dlt

```diff
@@ -32,3 +32,142 @@ FEATURES_CFG_DATA.Features.FusaEnable                       | 0
+GPIO_CFG_DATA.GpioPinConfig1_GPP_A12 | 0x020C2001
+GPIO_CFG_DATA.GpioPinConfig0_GPP_A13 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_A14 | 0x0350A385
+GPIO_CFG_DATA.GpioPinConfig0_GPP_A16 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_A23 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S00 | 0x0350A383
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S01 | 0x0350A383
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S02 | 0x0350A383
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S03 | 0x0350A383
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S04 | 0x0350A385
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S05 | 0x0350A385
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S06 | 0x0350A385
+GPIO_CFG_DATA.GpioPinConfig0_GPP_S07 | 0x0350A385
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H00 | 0x0350A281
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H01 | 0x0350A281
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H05 | 0x0550A583
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H10 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H11 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H12 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H13 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H14 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H15 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H19 | 0x0350A383
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H20 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H21 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_H22 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D00 | 0x0550A581
+GPIO_CFG_DATA.GpioPinConfig1_GPP_D00 | 0x08002019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D01 | 0x0550A381
+GPIO_CFG_DATA.GpioPinConfig1_GPP_D01 | 0x08012019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D02 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D03 | 0x0550A581
+GPIO_CFG_DATA.GpioPinConfig1_GPP_D03 | 0x08032019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D04 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D05 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D06 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D07 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D08 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D16 | 0x0350A381
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D17 | 0x0550E281
+GPIO_CFG_DATA.GpioPinConfig1_GPP_D17 | 0x08112019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D18 | 0x0550A581
+GPIO_CFG_DATA.GpioPinConfig1_GPP_D18 | 0x08122019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_D19 | 0x0350A383
+GPIO_CFG_DATA.GpioPinConfig0_GPD06 | 0x0750A281
+GPIO_CFG_DATA.GpioPinConfig0_GPD07 | 0x0750A281
+GPIO_CFG_DATA.GpioPinConfig0_GPD09 | 0x0750A281
+GPIO_CFG_DATA.GpioPinConfig0_GPP_C02 | 0x0350A281
+GPIO_CFG_DATA.GpioPinConfig0_GPP_C05 | 0x0350A281
+GPIO_CFG_DATA.GpioPinConfig0_GPP_C08 | 0x0550A583
+GPIO_CFG_DATA.GpioPinConfig1_GPP_C08 | 0x0B082019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_C09 | 0x0550A583
+GPIO_CFG_DATA.GpioPinConfig1_GPP_C09 | 0x0B092019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_C10 | 0x0550A583
+GPIO_CFG_DATA.GpioPinConfig1_GPP_C10 | 0x0B0A2019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_C11 | 0x0550A583
+GPIO_CFG_DATA.GpioPinConfig1_GPP_C11 | 0x0B0B2019
+GPIO_CFG_DATA.GpioPinConfig0_GPP_C13 | 0x0350A381

```

Comment:
GPP_C12/C13 should not be skipped.   So it needs to have GpioPinConfig1 overriding in DLT file.